python-lib/nlp_utils.py

```diff
@@ -3,16 +3,45 @@
 import unicodedata
 
 
-def get_keyword(text: AnyStr, normalize_case: bool) -> AnyStr:
-    """Return text in its wanted-case form"""
-    return text.lower() if normalize_case else text
+def normalize_case_text(text: AnyStr, lowercase: bool) -> AnyStr:
```

Comment:
I'd rather not, since the call of normalize_case_text will be combined with the unicode_normalization in the next PR 
(sthg like 

```
unicode_normalize(
                     text=normalize_case_text(pattern,self.normalize_case),
                     normalize_diacritics=self.normalize_diacritics
)
```
) 
and I didn't want to do two steps as I am using a comprehension list for now

Is it okay to keep it as a one-liner anyway?

Reply:
Okay, then maybe don't use the word `normalize` here as it could induce some confusion with unicode normalization. Maybe something like `def lower_if(text: AnyStr, lower: bool)` ?

Reply:
I understand the confusion, thanks, I've changed it here 883b305